### ISSUE TITLE
Error related to Product publications and revision control

### ISSUE THREAD

**Issue by mikemurray** [reactioncommerce/reaction#2079]
Sometimes we get the following error in terminal because of collection observables across the Media / Product / Products subscriptions.

```
Exception in queued task: Error: Could not find element with id NCHtKw9CaT5WssF4s to change
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:188:13)
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:403:10)
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:1262:19)
    at Object.Revisions.find.observe.changed (server/publications/collections/products.js:346:20)
    at [object Object].observeChangesCallbacks.changed (packages/minimongo/observe.js:162:28)
    at self.applyChange.changed (packages/minimongo/observe.js:66:44)
    at packages/mongo/observe_multiplex.js:182:30
    at Array.forEach (native)
    at Function._.each._.forEach (packages/underscore.js:139:11)
    at Object.task (packages/mongo/observe_multiplex.js:176:9)
    at [object Object]._.extend._run (packages/meteor.js:807:18)
    at packages/meteor.js:785:14
```

We should try to fix this, once and for all. Some suggestions follow.

- Remove old, Blaze, PDP page **(should be done regardless)**
- Maybe disallow revision control from being disabled completely
- Get rid of collection observables in publications in favor of doing that on client side, return multiple cursors from those publications instead.
- GraphQL / Catalog collection to piece things together
- Change how revision control works

### Expected behavior

Should never happen

### Actual Behavior

Happens infrequently, but still seems to be an issue

### Steps to Reproduce the Behavior

??? infrequently reproducible

**Comment by aaronjudd** [comment]
Let's create an issue for removing the PDP.

**Comment by spencern** [comment]
@mikemurray does this issue still exist?

**Comment by zenweasel** [comment]
@spencern I still see it regularly. But not in any reproducible way.

**Comment by zenweasel** [comment]
I'm debating whether to close this since it's not actionable but keeping open so that we can have the history so someone doesn't just open a new issue with the same error.

**Comment by aldeed** [comment]
Fixed in 1.9.0
